sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -830,6 +844,16 @@ struct _pi_context : _pi_object {
   }
 };
 
+struct ze_event {
+  // Level Zero event handle.
+  ze_event_handle_t Handle;
+
+  // Level Zero event pool handle.
+  ze_event_pool_handle_t Pool;
+
+  bool HostVisible = false;
+};
```

Comment:
Why do you need this data structure? It looks like a subset of a _pi_event, so why not just use the later?

Reply:
Idea of the patch was avoiding creation/deletion of pi_event objects for discarded events.  This structure is lightweight (it contains only two pointers and a flag) while pi_event contains more information which is unnecessary in this case.

Reply:
It'd be good to reuse PI events, but I don't like the idea to create parallel structures just for the reason of being more "lightweight" where it wasn't proved to be a problem. Even if we had to go this route then this structure should be re-used by the PI event.

Reply:
I agree with your concern and removed introduced ze_event data structure and now use pi_event instead.